service/src/main/java/bio/terra/workspace/common/utils/AwsUtils.java

```diff
@@ -701,6 +692,40 @@ public static NotebookInstanceStatus getSageMakerNotebookStatus(
     }
   }
 
+  /**
+   * Start a AWS SageMaker Notebook
+   *
+   * @param awsCredentialsProvider {@link AwsCredentialsProvider}
+   * @param notebookResource {@link ControlledAwsSageMakerNotebookResource}
+   */
+  public static void startSageMakerNotebook(
```

Comment:
Is this used for anything?

Reply:
yes, in undo of STOP step. 
(This too should be leverage from CRL later)